tests/Microsoft.Identity.Test.Unit/TelemetryTests/OTelInstrumentationTests.cs

```diff
@@ -70,7 +70,7 @@ public async Task AcquireTokenOTelTestAsync()
                 await AcquireTokenMsalClientExceptionAsync().ConfigureAwait(false);
 
                 s_meterProvider.ForceFlush();
-                VerifyMetrics(5, _exportedMetrics, 2, 2);
+                VerifyMetrics(6, _exportedMetrics, 2, 2);
```

Comment:
Since this is without the extension, the metrics captured here will stay 5. This is the reason for test failures.